config/config_schema.json

```diff
@@ -341,6 +341,143 @@
+                    "type": "string",
+                    "examples": [
+                        "/etc/carta/userlookup.txt"
+                    ]
+                },
+                "groupsField": {
+                    "description": "Name of field containing list of user roles/groups",
+                    "type": "string",
+                    "examples": [
+                        "groups",
+                        "roles"
+                    ]
+                },
+                "requiredGroup": {
+                    "description": "Role to ensure is included among the values in groupsField",
+                    "type": "string",
+                    "examples": [
+                        "carta-users",
+                        "carta-testers"
+                    ]
+                },
+                "localPublicKeyLocation": {
+                    "description": "Path to public key (in PEM format) used for verifying JWTs",
+                    "type": "string",
+                    "examples": [
+                        "/etc/carta/carta_public.pem"
+                    ]
+                },
+                "localPrivateKeyLocation": {
+                    "description": "Path to private key (in PEM format) used for signing JWTs",
+                    "type": "string",
+                    "examples": [
+                        "/etc/carta/carta_private.pem"
+                    ]
+                },
+                "keyAlgorithm": {
+                    "$ref": "#/definitions/keyAlgorithm",
+                    "default": "RS256"
+                },
+                "issuer": {
+                    "description": "Issuer field for JWT",
+                    "type": "string",
+                    "examples": [
+                        "my-carta-server"
+                    ]
+                },
+                "cacheAccessTokenMinValidity": {
+                    "description": "If previously issued an access token from upstream server with at least this many seconds of lifetime remaining, a local token with the previous token's remaining lifetime will be issued instead of performing an upstream query",
+                    "type": "integer",
+                    "default": 100
+                },
+                "symmetricKeyLocation": {
+                    "description": "Path to symmetric key (base64-encoded) used for refresh tokens. At present this uses the A256GCM algorithm which requires 32 bytes of random data which can be generated using `openssl rand -base64 32`",
+                    "type": "string",
+                    "examples": [
+                        "/etc/carta/carta_symmetric.pem"
+                    ]
+                },
+                "symmetricKeyType": {
+                    "description": "As per options at https://www.iana.org/assignments/jose/jose.xhtml when using direct use of shared key",
```

Comment:
"when using direct use of shared key" sounds odd. Should this be "when a shared key is used directly"?

The description also doesn't describe the option. Should this be "Type of symmetric key as per (...)"

Is the "shared key" the same thing as the "symmetric key"? Is the "shared key" qualification also needed for the location option above?